module/applications/item/item-sheet.mjs

```diff
@@ -500,6 +501,70 @@ export default class ItemSheet5e extends ItemSheet {
       return this.item.update({"system.damage.parts": damage.parts});
     }
   }
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  _onDragStart(event) {
+    const li = event.currentTarget;
+    if ( event.target.classList.contains("content-link") ) return;
+
+    // Create drag data
+    let dragData;
+
+    // Active Effect
+    if ( li.dataset.effectId ) {
+      const effect = this.item.effects.get(li.dataset.effectId);
+      dragData = effect.toDragData();
+    }
+
+    if ( !dragData ) return;
```

Comment:
```suggestion
    const effect = this.item.effects.get(li.dataset.effectId);
    const dragData = effect?.toDragData();
    if ( !dragData ) return;
```

Reply:
I believe the `if` is there to handle future use cases where other items such as advancements are also draggable.